microprofile-config/README.adoc

```diff
@@ -11,6 +11,7 @@ The `microprofile-config` quickstart demonstrates the use of the MicroProfile Co
 :standalone-server-type: default
 :archiveType: jar
 :archiveName: {artifactId}
+:arq-prerequisities: These tests require that {jbossHomeName} environment variable is set.
```

Comment:
Why not `JBOSS_HOME` instead of {jbossHomeName}, as used in run-arquillian-tests.adoc bottom note? The way I see jbossHomeName is just a prod vs community differentiation for the app server distribution directory, and should not be confused with the env property JBOSS_HOME.

Reply:
The tests require JBOSS_HOME, it can't be EAP_HOME which would be in jbossHomeName. I can change it to JBOSS_HOME env var OR jbossHome sys prop but that seems too much information and not much value. Should I change it?

Reply:
I mean this line to be instead:
:arq-prerequisities: These tests require that `JBOSS_HOME` environment variable is set.

Reply:
thanks, I understand now